nodes/usb_cam_node.cpp

```diff
@@ -343,11 +366,28 @@ class UsbCamNode
     ros::Rate loop_rate(this->framerate_);
     while (node_.ok())
     {
+      ros::Duration last_update_dur = ros::Time::now() - last_cam_update_time;
+      camera_status.name = "USB Camera Device Monitoring";
+      camera_status.hardware_id = "None";
+      if (last_update_dur > ros::Duration(loop_rate))
+    {
+      camera_status.level = diagnostic_msgs::DiagnosticStatus::ERROR;
+      camera_status.message = "USB Camera is not publishing Images.";
+    }
+    else
+    {
+      camera_status.level = diagnostic_msgs::DiagnosticStatus::OK;
+      camera_status.message = "Normal";
+    }
```

Comment:
i dont really see why this part is necessary. it seems like it duplicates the frequency status below

Reply:
That's true, I have removed it now. We thought we wanted a separate messaged key value pair for the Datadog Logger.